tests/ts/lslogins/checkuser

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+TS_TOPDIR="${0%/*}/../.."
+TS_DESC="read logins"
+
+. "$TS_TOPDIR"/functions.sh
+ts_init "$*"
+ts_check_test_command "$TS_CMD_LSLOGINS"
+
+#
+# lslogin
+#
+$TS_CMD_LSLOGINS --help|grep "\--lastlog2" >/dev/null
+if [ $? -eq 0 ]; then
+    # tests for lastlog2 environment
+    $TS_CMD_LSLOGINS -L --lastlog2="${0%/*}/lastlog2.db" root |grep "test-tty" >/dev/null
```

Comment:
Instead of grepping you could redirect the output to `$TS_OUTPUT` and the testing framework will automatically check that the correct output has been generated.

Reply:
Yes, done.